velero-csi-plugin/volumesnapshotcontents_restorer.go

```diff
@@ -32,42 +33,60 @@ type VSCRestorer struct {
 	log logrus.FieldLogger
 }
 
+// AppliesTo returns information about which resources the VSCRestorer action should be invoked for.
+// VSCRestorer RestoreItemAction plugin's Execute function will only be invoked on items that match the returned
+// selector. A zero-valued ResourceSelector matches all resources.
 func (p *VSCRestorer) AppliesTo() (velero.ResourceSelector, error) {
 	return velero.ResourceSelector{
 		IncludedResources: []string{"volumesnapshotcontent.snapshot.storage.k8s.io"},
 	}, nil
 }
 
+// Execute allows the RestorePlugin to perform arbitrary logic with the item being restored,
```

Comment:
I would drop "allows the RestorePlugin to perform arbitrary logic" bit and just document *this* plugin's behavior. I would also try to be more specific about what this plugin does, along the same lines of the comment for the VolumeSnapshot restorer.